assets/js/base/context/hooks/use-checkout-submit.js

```diff
@@ -31,11 +32,16 @@ export const useCheckoutSubmit = () => {
 	const waitingForProcessing =
 		isProcessing || isAfterProcessing || isBeforeProcessing;
 	const waitingForRedirect = isComplete && ! hasError;
+	const defaultLabel =
+		paymentMethod.placeOrderButtonLabel ||
+		__( 'Place Order', 'woo-gutenberg-products-block' );
+	const label = __experimentalApplyCheckoutFilter( {
+		filterName: 'placeOrderLabel',
```

Comment:
should this be `'placeOrderButtonLabel'` to keep it consistent with all other places we refer to this?

Reply:
The filter should indeed be called `placeOrderButtonLabel` instead of `placeOrderLabel`. I also adjusted this one.